src/bottomsheet/bottomsheet.android.ts

```diff
@@ -155,7 +157,18 @@ function initializeBottomSheetDialogFragment() {
                 }, 0);
             }
 
-            
+            if (this._dismissOnDraggingDownSheet !== undefined) {
```

Comment:
see last comment this should be :
```ts 
if (this._dismissOnDraggingDownSheet)
```

Reply:
This is also implemented the same way than ````transparent```.
Icould changed this or both parameters if you want.